src/products/components/ProductCreatePage/form.tsx

```diff
@@ -306,13 +306,14 @@ function useProductCreateForm(
   const submit = () => onSubmit(data);
 
   const disabled =
```

Comment:
Lets make a function of it, it was hardly readable before and it's only more so now